libnetwork/drivers/bridge/internal/rlkclient/rootlesskit_client_linux.go

```diff
@@ -128,10 +139,11 @@ func (c *PortDriverClient) AddPort(
 		// even when network driver is set to "slirp4netns".
 		//
 		// Most users are using "builtin" port driver and will not see this warning.
-		return nil, fmt.Errorf("protocol %q is not supported by the RootlessKit port driver %q, discarding request for %q",
-			proto,
-			c.portDriverName,
-			net.JoinHostPort(hostIP.String(), strconv.Itoa(hostPort)))
+		err := &ProtocolUnsupportedError{
+			apiProto:       apiProto,
+			portDriverName: c.portDriverName,
+		}
+		return nil, err
```

Comment:
Related to that; I see at the start of this function we have logic for if there's no portdriverclient, in which case we return a stub remover func;

```
return func() error { return nil }, nil
```


(again, just exploring what would make "we tried to map the port, weren't able to, but it's ok to skip this port" fit into existing logic)